### ISSUE TITLE
Unsupported URL Type: github:kevva/gulp-decompress#... when installing generator-fluxible

### ISSUE THREAD

**Issue by jabbermarky** [yahoo/generator-fluxible#46]
on OSX 10.11
node v0.12.7
npm v2.11.3

I can't install the generator on my OS X box.
any suggestions?

Mark

**Comment by mridgway** [comment]
That package doesn't seem to be part of the generator. Did you add that to the package.json yourself?

**Comment by jabbermarky** [comment]
No, I followed the simple instructions from the flexible.io Quick Start page : npm install -g yo generator-fluxible

**Comment by mridgway** [comment]
That's very odd. Maybe there is another package that you have installed globally that is trying to install `gulp-compress`? Neither `yo` or `generator-fluxible` should attempt to install that package. Running that command works for me locally.

**Comment by jabbermarky** [comment]
that was it. I removed several global packages that were listing errors in calls to npm list and then I was able to install yo and generator-flexible successfully. thanks for your help!
On Oct 19, 2015, at 4:48 PM, Michael Ridgway <notifications@github.com> wrote:

That's very odd. Maybe there is another package that you have installed globally that is trying to install gulp-compress? Neither yo or generator-fluxible should attempt to install that package. Running that command works for me locally.
